### PR TITLE
60 create store button

### PR DESCRIPTION
**Pull request by emadsu**
I added a plain store button that doesn't go anywhere but it says that it's coming soon so give it a chance guys!

### CODE REVIEW

**Comment by MangoMagoCane** [comment]
Make sure to merge with the lastest commit from main

File: frontend/src/game/scenes/MainMenu.ts [hunk 1]
```diff
@@ -1,44 +1,66 @@
-import { GameObjects, Scene } from 'phaser';
-import { EventBus } from '../EventBus';
+import { GameObjects, Scene } from "phaser";
```
Comment (jvgusch) [inline frontend/src/game/scenes/MainMenu.ts:1]: Your Prettier settings might be messed up, it's changing things against the style the rest of the team is submitting. You might want to double-check it's working correctly.

**Review by jvgusch — CHANGES_REQUESTED** [review]
Everything looks great so far, but I think I'd like to actually see the functionality as described in #60 as opposed to the placeholder you have right now. Unless there's some conversation I missed where this button is holding up others, you have time to create the user info page as well. It doesn't have to be fully fledged, but it can at least display the same coming soon message you already have.

**Comment by josephmaxwellhyland** [comment]
We will be refactoring these buttons

**Pull request closed without merging**